### PR TITLE
Fill in changelog and bump version for 0.25.3

### PR DESCRIPTION
**Pull request by Shnatsel**
Let's get the work we did on image orientation into the hands of users!

### CODE REVIEW

File: CHANGES.md
Comment (fintelia) [inline CHANGES.md:22]: The "actually removes rayon from the dependency tree" part isn't true yet. https://github.com/johannesvollmer/exrs/pull/241 also needs to land first (tracked by https://github.com/image-rs/image/issues/2028)

Comment (fintelia) [reply CHANGES.md:22]: And actually, don't we need to change our Cargo.toml to enable ravif's threading feature when our rayon feature _is_ enabled?

**Comment by fintelia** [comment]
The one thing I've been second guessing is whether `Orientation` should be a top-level type. I've been wanting to reduce the number of top level types, and we probably will be adding a `metadata` module to group all the other metadata related types we add, so I think we it might be good to place `Orientation` there to start with

File: CHANGES.md
Comment (Shnatsel) [reply CHANGES.md:22]: We do, and in fact the latest `ravif` with `default-features=false`  will not enable rayon by default anymore, so this release is needed just to preserve the previous default behavior when using the latest `ravif`.

I've opened https://github.com/image-rs/image/pull/2348 for it

**Comment by Shnatsel** [comment]
I've opened https://github.com/image-rs/image/pull/2349 to relocate `Orientation`

**Comment by Shnatsel** [comment]
I've rewritten the changelog entry for rayon. I don't think we should wait on the exr change to ship this, since it appears to be a breaking change, the maintainer is busy, and so there is no clear timeline for it.

**Review by HeroicKatora — APPROVED** [review]

**Comment by HeroicKatora** [comment]
@fintelia If you agree with preparing `0.26` next, feel free to merge #2344 after the merge of the tagged release commit.

**Comment by fintelia** [comment]
@HeroicKatora Sorry I've been meaning to respond in that other thread. [I wrote up some thoughts here](https://github.com/image-rs/image/issues/2316), but rather than working on 0.26, I actually think that we should old off starting a major release for a little bit and then make a _1.0_ release.

**Review by fintelia — APPROVED** [review]
Might be worth adding that this release brings substantially improved lossless WebP decoding performance. I measured it being between 2x and 2.5x faster!

(We usually don't mention codec improvements because they tend not to be tied to image releases, but these specific optimizations were bundled with the `image-webp` 0.2.0 release)

**Comment by Shnatsel** [comment]
The changelog still mentions #2348 which hasn't landed yet. So please don't publish a release just yet!

I'm hoping we could merge https://github.com/kornelski/cavif-rs/pull/85 soon and fix the whole issue around MSRV. If it's not merged in a couple of days, we can pin an older `ravif` and then ship.

**Comment by Shnatsel** [comment]
Ah, nevermind, too late :sweat_smile: 

I've opened https://github.com/image-rs/image/pull/2352 to amend the changelog.

**Pull request merged**
